### PR TITLE
Process data and new file

### PR DESCRIPTION
**Pull request by t1a2l**
Added a csv file for testings. Added a function for processing the data in an object array.

### CODE REVIEW

File: app.html [hunk 1]
```diff
@@ -6,25 +6,45 @@
 	<div id="content"></div>
 
 	<script>
-	  function handleFileSelect(evt) {
-		var files = evt.target.files; // FileList object
+		let myFile;
```
Comment (oferb) [inline app.html]: Can you format the file with 2 spaces indentation instead of tabs?

File: app.html [hunk 2]
```diff
@@ -6,25 +6,45 @@
...
-	  document.getElementById('files').addEventListener('change', handleFileSelect, false);
+
+		function processData(data) {	
+			let csv_lines = data.split("\n");
+			let objectNames =  csv_lines[0].split(",");
+			let entries = [];
+			for(let i = 1; i < csv_lines.length; i++){
+				entry = csv_lines[i].split(",");
+				entries[i-1] = new Object();
+				for(let j = 0; j < entry.length; j++){
```
Comment (oferb) [inline app.html]: I think it will be clearer to rename `j` to `fieldIndex` or `columnIndex`.

File: app.html [hunk 3]
```diff
@@ -6,25 +6,45 @@
...
 		}
-	  }
 
-	  document.getElementById('files').addEventListener('change', handleFileSelect, false);
+
+		function processData(data) {	
+			let csv_lines = data.split("\n");
+			let objectNames =  csv_lines[0].split(",");
+			let entries = [];
+			for(let i = 1; i < csv_lines.length; i++){
```
Comment (oferb) [inline app.html]: Please add a space here: `for(->for (`. 
Also below.

Also here: `){->) {`

File: app.html (hunk 2 above)
Comment (t1a2l) [reply app.html]: done

**Pull request merged**